libs/features/pages/consultancy-page/src/lib/Component/elewa-website-learning-services/elewa-website-learning-services.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+import { __learningDevicesData } from '@elewa-website/data/sections';
+import { InfoCard } from '@elewa-website/models/schema/ui/cards';
+
+@Component({
+  selector: 'elewa-website-learning-services',
+  templateUrl: './elewa-website-learning-services.component.html',
+  styleUrls: ['./elewa-website-learning-services.component.scss'],
+})
+export class ElewaWebsiteLearningServicesComponent {
+  @Input() cardsData!: InfoCard[];
```

Comment:
As this is not reusable, the 'cardsData' variable should be a normal variable instead of an input variable